### DISCUSSION TITLE
Can a K8S cluster join two different Kubevela management and control clusters at the same time?

### DISCUSSION THREAD

**Discussion by skyshaokai** [kubevela/kubevela#4941] [Q&A] [answered]
Can a K8S cluster join two different Kubevela management and control clusters at the same time?

We hope to build two sets of kubevela clusters to isolate the online and offline environment. Set up a set of kubevela control cluster kubevela online in the online environment, and set up a set of kubevela control cluster kubevela offline in the offline environment. Can the controlled K8S cluster join the kubevela online and kubevela offline clusters at the same time?
for example : 
Can the Managed cluster B  manged by Hub cluster online and Hub  cluster offline at same time ？

<img width="1081" alt="image" src="https://user-images.githubusercontent.com/11512967/198276582-fc859577-754b-48e3-9f0c-55037a94e29a.png">

**Comment by wonderflow** [comment, marked as answer]
Yes, this is possible. KubeVela don't rely on the managed cluster to be exclusive, and it don't put any control plane pods inside the managed cluster.

One thing you should keep in mind is by default, KubeVela will keep reconcile the resource to the desired state, please make sure the resources are not conflict by two controlled applications the same time. You can configure [apply-once policy](https://kubevela.net/docs/end-user/policies/apply-once) if you don't want to the reconcilation.

**Reply by skyshaokai** [reply]
our resource in different namespace and different managed cluster manage different  namespace。
